superset-frontend/src/explore/components/SaveModal.tsx

```diff
@@ -518,3 +522,7 @@ function mapStateToProps({
 }
 
 export default withRouter(connect(mapStateToProps)(SaveModal));
+
+// todo(hughhhh): For testing
+// - need to revisit once we convert this to functional component
```

Comment:
I think you can remove the todo and just say that you're exporting this for testing.